packages/nimbus/src/components/toggle-button-group/toggle-button-group.mdx

```diff
@@ -12,223 +12,442 @@ menu:
+        <Icons.FormatBold />
+      </ToggleButtonGroup.Button>
+      <Tooltip>Bold</Tooltip>
+    </TooltipTrigger>
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="format-italic" aria-label="Italic">
+        <Icons.FormatItalic />
+      </ToggleButtonGroup.Button>
+      <Tooltip>Italic</Tooltip>
+    </TooltipTrigger>
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="format-underline" aria-label="Underline">
+        <Icons.FormatUnderlined />
+      </ToggleButtonGroup.Button>
+      <Tooltip>Underline</Tooltip>
+    </TooltipTrigger>
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="format-strikethrough" aria-label="strikethrough">
+        <Icons.StrikethroughS />
+      </ToggleButtonGroup.Button>
+      <Tooltip>Strikethrough</Tooltip>
+    </TooltipTrigger>
+    <TooltipTrigger delay={0} closeDelay={0}>
+      <ToggleButtonGroup.Button id="options" aria-label="options">
+        <Icons.MoreVert />
+      </ToggleButtonGroup.Button>
+      <Tooltip>More Options</Tooltip>
+    </TooltipTrigger>
+  </ToggleButtonGroup.Root>
+);
 ```
 
-## Accessibility
+> [!CAUTION]\
+> **Don't**
+>
+> -   Do not mix unexpected and unrelated actions within toolbars.
+> -   Do not mix button styles, mixing styles can be confusing and alter hierarchy for the user.
+> -   Choices that need different tones may not be suitable for button group usages.
+> -   It is uncommon to select more than one action in a button grouping. Rich text styling is an example where this is loosened, and users may select to style text with multiple styles at once.
 
-The ToggleButtonGroup component is built with accessibility in mind:
+```jsx-live
+const App = () => (
+    <ToggleButtonGroup.Root aria-label="Mixed Actions (Bad Example)" selectedKeys={["bad-italic"]} tone="primary">
+        <ToggleButtonGroup.Button id="bad-bold" aria-label="Bold">
+          <Icons.FormatBold />
+        </ToggleButtonGroup.Button>
+        <ToggleButtonGroup.Button id="bad-italic" aria-label="Italic">
+          <Icons.FormatItalic /> Italics
+        </ToggleButtonGroup.Button>
+            <ToggleButtonGroup.Button id="bad-underline" aria-label="Underline">
+      <Icons.FormatUnderlined />
+    </ToggleButtonGroup.Button>
+                <ToggleButtonGroup.Button id="bad-save" aria-label="Save">
+      <Icons.Save /> Save
+    </ToggleButtonGroup.Button>
+                  <ToggleButtonGroup.Button id="bad-cancel" aria-label="Cancel" colorPalette="error">
+      <Icons.Clear /> Cancel
+    </ToggleButtonGroup.Button>
+    </ToggleButtonGroup.Root>
+);
+```
 
-- Implements proper ARIA roles (uses `radiogroup` and `radio` roles)
-- Supports keyboard navigation using arrow keys
-- Provides appropriate focus management
-- Requires an `aria-label` for the group to ensure screen reader users
-  understand its purpose
+## Properties
```

Comment:
@commercetools/first-contact-team-fe Should we stick to `Specs` or `Properties`? We seem to use them interchangeably in most of the docs.

Reply:
I discussed this with @ellelevy and we agreed that `Specs` was preferred, but only when we get around to adding developer documentation, so `Properties` should suffice for now 👍